barbarik.py

```diff
@@ -577,9 +577,12 @@ def one_experiment(self, experiment, j, i):
         if self.thresholdSolutions < self.minSamples:
             return None, None
 
+        # generate a new seed value for every call to the sampler
+        preSeed = random.randint(1,j*1000)
+        newSeed = random.randint(preSeed, i*j*1000)
```

Comment:
What's this? You don't need this at all. Just `newSeed = random.randint(1, X)`. What's the point of j here? What's the point of preSeed? None of this makes sense.